appserver/ejb/ejb-container/pom.xml

```diff
@@ -276,11 +276,5 @@
             <artifactId>payara-api</artifactId>
```

Comment:
Missing copyright headers in most of these poms!

Reply:
I updated the pom headers now as well.